experiments/tango-in-beaker.yml

```diff
@@ -9,30 +9,27 @@ executor:
   type: beaker
   # Change this to the Beaker workspace you want to use.
   beaker_workspace: ai2/task-complexity
-  #beaker_image: dirkg/catwalk
+  beaker_image: dirkg/catwalk
   # You probably want to leave this as 'base' unless you have your own image
   # that already has a conda environment under a different name.
   venv_name: base
   # These are the Beaker clusters that you want to use to run steps.
-  # Tango will inspect all of the clusters in this list at runtime and pick
-  # the first one with enough free resources for your jobs.
   clusters:
```

Comment:
If this is basically just a list of all on-prem clusters, you could omit it. Beaker allows that now as long as the job is preemptible.

Reply:
As in, I don't have to specify the list of clusters at all anymore?